tobago-example/tobago-example-demo/pom.xml

```diff
@@ -39,7 +39,10 @@
     <arquillian.browser>phantomjs</arquillian.browser>
     <selenium.version>3.7.1</selenium.version>
     <tomee.version>7.0.4</tomee.version>
+    <meecrowave.version>1.2.1-SNAPSHOT</meecrowave.version>
```

Comment:
I think it should be the 1.2.1 version.

Reply:
yes indeed! 👍 
tomee is now 7.0.5 and johnzon 1.1.7
Preparing for JavaLand and have to finish $$dayjob stuff, so  sorry for not finding time to polish further :(

Reply:
No problem, I'll take care of this (as discussed on the owb user list). Thanks.